README.md

```diff
@@ -1 +1,88 @@
-# SwarmForm
\ No newline at end of file
+# SwarmForm
+
+SwarmForm is a distributed workflow management system for High Performance Computing environments. SwarmForm introduces task clustering to reduce the makespan of workflows executed using it by minimizing the overheads. SwarmForm follows a distributed architecture with independent programs controlling different functional layers in workflow management.
+## Getting Started
+
+These instructions will get you familiar with the installation and basic functions of the SwarmForm.
+
+### Prerequisites
+
+What things you need to install to run the SwarmForm.
```

Comment:
Better to rephrase as "Following prerequisites should be installed to run SwarmForm"